spec/features/account_connected_apps_spec.rb

```diff
@@ -86,19 +93,25 @@
 
     expect(page).to have_field(user.email) { |field| !field[:checked] }
```

Comment:
Could we include some coverage here for the client-side validation we're expecting to see? We might need to enable JavaScript for it to test client-side validation (add `:js` metadata to the `scenario`), though technically I'd expect it to work similarly with JavaScript disabled.

Something like this would work, and ensure that focus moves as expected and the error message is accessible to assistive technology:

```suggestion
    expect(page).to have_field(user.email) { |field| !field[:checked] }

    click_on t('help_text.requested_attributes.change_email_link')
    input = page.find(':focus')
    expect(input).to have_name(strip_tags(t('help_text.select_preferred_email_html', sp: identity.display_name)))
    expect(input).to have_description(t('simple_form.required.text'))

```